packages/calling-component-bindings/src/getRemoteParticipantsExcludingConsumers.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { createSelector } from 'reselect';
+import { getRemoteParticipants } from './baseSelectors';
+
+/**
+ * @private
+ */
+export const getRemoteParticipantsExcludingConsumers = createSelector([getRemoteParticipants], (remoteParticipants) => {
+  /* @conditional-compile-remove(rooms) */
+  {
+    const participants = Object.values(remoteParticipants ?? {});
+    return participants.filter((remoteParticipants) => remoteParticipants.role !== 'Consumer');
```

Comment:
I think my code snippet was poor, this will correctly return an array of remoteParticipants who aren't consumers, but it won't return the same as the conditional-compile return `return remoteParticipants` which is a key map of ids to remote participants

Reply:
It worked when I tested it locally. Will change it back to key map of ids to be sure.

Reply:
Yeah I'm surprised/confused nothing broken in the build and beta and stable tests - but they were returning two different interfaces right?